### PR TITLE
Update of docs after tests on bare metal

### PR DESCRIPTION
**Pull request by ksankiew**

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Connection aborted.', ConnectionRefusedError(111, 'Connection refused')'
+
+```console
+./api_call.sh create t
+Create tenant
+Please provide tenant name abbb
+Please provide scope (group name) sun
+{"title": "Unexpected error occurred: ('Connection aborted.', ConnectionRefusedError(111, 'Connection refused'))"}
+```
+Above message can indicate on MinIO -- Management Api connections issues, credentials or wrong port included in minio-access-info kubernetes secret.
```
Comment (mareklew-Intel) [inline docs/troubleshooting.md]:
```suggestion
Above message can indicate on Minio -- Management Api connections issues, credentials or wrong port included in minio-access-info kubernetes secret.
```
In others documents we use name like "Minio" not MinIO.

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Troubleshooting
+
+## Typical errors
+
+An error occurred during namespace creation: Unauthorized
```
Comment (mareklew-Intel) [inline docs/troubleshooting.md]: In my opinion it`s will be more visible if you put there subsection symbol '###'.

In other errors too.

File: docs/nodeport.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+## Kubernetes Node Port
```
Comment (dtrawins) [inline docs/nodeport.md]: I suggest to include info specific for nginx service exposed via node port in the dex helm documentation.
It looks like whole content is specific to dex deployment only

File: docs/privateregistry.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+## Private Docker Registry
```
Comment (dtrawins) [inline docs/privateregistry.md]: Add a link to this doc from the requirements section related to docker registry.

File: docs/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Troubleshooting
```
Comment (dtrawins) [inline docs/troubleshooting.md]: Add a link to this section from the deployment guide and installation quicksheet.

File: docs/troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+## Typical errors
+
+### An error occurred during namespace creation: Unauthorized
+
+```console
+./api_call.sh c t
```
Comment (pawel-pieniazek) [inline docs/troubleshooting.md]: in instruction for user it is better to use whole name - like `create tenant`

File: docs/troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+Please provide tenant name abbb
+Please provide scope (group name) sun
+{"title": "An error occurred during namespace creation: Unauthorized"}
+```
+Above error indicates on wrong connection information in kubeapi server config file, most probably wrong oidc issuer url or wrong port in issuer url.
+
+### Connection aborted.', ConnectionRefusedError(111, 'Connection refused')'
+
+```console
+./api_call.sh create t
```
Comment (pawel-pieniazek) [inline docs/troubleshooting.md]: same as above

File: docs/troubleshooting.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+Please provide tenant name abbb
+Please provide scope (group name) sun
+{"title": "Unexpected error occurred: ('Connection aborted.', ConnectionRefusedError(111, 'Connection refused'))"}
+```
+Above message can indicate on Minio -- Management Api connections issues, credentials or wrong port included in minio-access-info kubernetes secret.
+
+### Unexpected error occurred: Failed to parse: minioplatform.default:9000\n
+
+```console
+./api_call.sh c t
```
Comment (pawel-pieniazek) [inline docs/troubleshooting.md]: same as above

File: docs/troubleshooting.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+Please provide tenant name abbb
+Please provide scope (group name) sun
+{"title": "Unexpected error occurred: Failed to parse: minioplatform.default:9000\n"}
+```
+minio.endpoint_url variable in minio-access-info was base64 encoded with additional newline sign. 
+
+### Unexpected error occurred: 'dict' object has no attribute 'to_dict'
+
+```console
+./api_call2.sh ls t
```
Comment (pawel-pieniazek) [inline docs/troubleshooting.md]: same as above

**Pull request merged**